src/components/Leaderboard.js

```diff
@@ -77,9 +79,15 @@ const Leaderboard = ({ content, limit = 100 }) => (
           emoji = ":3rd_place_medal:"
         }
         return (
-          <Item key={idx} to={`${githubUrl}${username}`}>
+          <Item key={idx} to={hasGitHub ? `${githubUrl}${username}` : "#"}>
             <ItemNumber>{idx + 1}</ItemNumber>
-            <Avatar src={`${githubUrl}${username}.png?size=40`} />
+            <Avatar
+              src={`${githubUrl}${username}.png?size=40`}
+              onError={(event) => {
```

Comment:
@minimalsm why is this `onError` needed? 🤔

Reply:
FYI I removed this. LMK if you think it's necessary.

Reply:
It was just a way to add a fallback image but this is better 👍